src/commands/lm/install.js

```diff
@@ -0,0 +1,43 @@
+const { flags: flagsLib } = require('@oclif/command')
+
+const Command = require('../../utils/command')
+const { installPlatform } = require('../../utils/lm/install')
+const { printBanner } = require('../../utils/lm/ui')
+
+class LmInstallCommand extends Command {
+  async run() {
+    const { flags } = this.parse(LmInstallCommand)
+
+    await this.config.runHook('analytics', {
+      eventName: 'command',
+      payload: {
+        command: 'lm:install',
+      },
+    })
+
+    try {
+      const installed = await installPlatform(flags.force)
+      if (installed) {
+        printBanner(this, flags.force)
+      }
+    } catch (error) {
+      this.log(error)
```

Comment:
```suggestion
      this.error(error)
```

So the error is printed on `stderr` and the exit code is non `0`.